x-pack/plugins/infra/public/pages/metrics/inventory_view/components/timeline/timeline.tsx

```diff
@@ -70,6 +70,7 @@ export const Timeline: React.FC<Props> = ({ interval, yAxisFormatter, isVisible
 
   const anomalyParams = {
     sourceId: 'default',
+    anomalyThreshold: source?.configuration.anomalyThreshold || '0',
```

Comment:
If we can import references to the anomaly thresholds from a constants file, that'd be more future-proof. I handled this in the anomaly alert PR by copy-pasting the anomaly severity levels into the `infra_ml` library (https://github.com/elastic/kibana/pull/89244/files#diff-b903eec3878be537efbc4914f9b0c4694ba1ac32ed12696b96d4cf0693bc600e)

(We probably want to import those constants directly from the ML plugin instead, but it's hard to do that without marking it as a "required" dep in `infra`, but we'll worry about that as a separate issue. If we have `infra_ml` re-define them for now, then we can just have it import and re-export them when we figure that out.)

Reply:
Agreed, I was thinking your PR would probably go in before mine and then I'd use those.

Reply:
Cool, if this one merges first I'll update these files in mine.